vyper/venom/passes/dft.py

```diff
@@ -18,9 +17,9 @@ def _process_instruction_r(self, bb: IRBasicBlock, inst: IRInstruction):
             bb.instructions.append(inst)
             return
 
-        for op in inst.get_inputs():
+        for op in inst.liveness:
             target = self.dfg.get_producing_instruction(op)
-            if target.parent != inst.parent or target.fence_id != inst.fence_id:
+            if target is None or target.parent != inst.parent or target.fence_id != inst.fence_id:
```

Comment:
when can target be None?

Reply:
Should not happen with frontend generated code, but can happen with user generated code. Upgraded the check to assert, and in the future when we can type in venom directly we should make this a compiler error.